src/pages/Landing/index.tsx

```diff
@@ -379,34 +337,6 @@ export default function Landing() {
   const location = useLocation()
   const queryParams = parse(location.search, { ignoreQueryPrefix: true })
 
-  const titles = useMemo(() => {
-    if (!originCountry) {
-      return {
-        header: null,
-        subHeader: null,
-      }
-    }
-
-    if (renderUkSpecificText) {
```

Comment:
So we are now blocking nft routes completely, instead of just rendering different text for them. Is that right?

Reply:
we are still rendering different text for them its just done where we were previously doing it ( in the render function)

Reply:
https://github.com/Uniswap/interface/pull/7489/files/f8ec4a9c481b0ef5f26a905deb41869ed8c61988#diff-f091cac6919a486258fcacdf80a2b93b0d4219d1aea881a10ad046b91b8c0be9R364-R367